### ISSUE TITLE
Rating Indicator re-write

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#4487]
The current implementation of the Rating Indicator does not follow the Fiori Screen Reader Specifications and should be re-implemented to have the appropriate aria attributes and roles.
